src/kokkos_ext/ArborX_DetailsKokkosExtStdAlgorithms.hpp

```diff
@@ -0,0 +1,154 @@
+/****************************************************************************
+ * Copyright (c) 2017-2022 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef ARBORX_DETAILS_KOKKOS_EXT_STD_ALGORITHMS_HPP
+#define ARBORX_DETAILS_KOKKOS_EXT_STD_ALGORITHMS_HPP
+
+#include <ArborX_DetailsKokkosExtAccessibilityTraits.hpp>
+#include <ArborX_Exception.hpp>
+
+#include <Kokkos_Core.hpp>
+
+namespace ArborX::Details::KokkosExt
+{
+
+template <typename ExecutionSpace, typename SrcView, typename DstView,
+          typename InitValueType = typename DstView::value_type>
+void exclusive_scan(ExecutionSpace const &space, SrcView const &src,
+                    DstView const &dst, InitValueType init = 0)
```

Comment:
There is no default value in `std::exclusive_scan `

Reply:
True. It would be annoying to write 0 all the time, though.

Reply:
If we go for alignment with the standard parallel algorithms with repeating the view name for in-place scans then I don't see how specifying the init value is a big deal.